src/services/time-slot.service.ts

```diff
@@ -0,0 +1,72 @@
+import { Injectable } from '@nestjs/common';
+import mongoose from 'mongoose';
+import { PatchingTimeSlotDTO } from 'src/dto/patching-time-slot.dto';
+import { TimeSlotDTO } from 'src/dto/time-slot.dto';
+import { TimeSlot } from 'src/models/time-slot.model';
+import { User } from 'src/models/user.model';
+import { GroupRepository } from 'src/repositories/group.repository';
+import { TimeSlotRepository } from 'src/repositories/time-slot.repository';
+
+@Injectable()
+export class TimeSlotService {
+  constructor(
+    private readonly timeSlotRepository: TimeSlotRepository,
+    private readonly groupRepository: GroupRepository,
+  ) {}
+
+  insertTimeSlot = async (
+    groupId: string,
+    parameters: TimeSlotDTO,
+  ): Promise<TimeSlot> =>
+    this.timeSlotRepository.insert({
+      group: new mongoose.Types.ObjectId(groupId),
+      ...parameters,
+    });
+
+  getAllGroupTimeSlots = async (groupId: string): Promise<TimeSlot[]> =>
+    this.timeSlotRepository.findManyBy({
+      group: new mongoose.Types.ObjectId(groupId),
+    });
+
+  getOneGroupTimeSlotById = async (timeSlotId: string): Promise<TimeSlot> =>
+    this.timeSlotRepository.findOneBy({ _id: timeSlotId });
+
+  updateOneGroupTimeSlotById = async (
+    conditions: {
+      groupId: string;
+      timeSlotId: string;
+    },
+    set: PatchingTimeSlotDTO,
+  ): Promise<boolean> =>
+    this.timeSlotRepository.updateOneBy(
+      {
+        group: new mongoose.Types.ObjectId(conditions.groupId),
+        timeSlotId: conditions.timeSlotId,
+      },
+      {
+        ...set,
```

Comment:
bad idea, `PatchingTimeSlotDTO` contains a `groupId` property that is not in TimeSlot model